frontend/README.md

```diff
@@ -144,8 +144,8 @@ If you made any changes to protos (see backend/README), you'll need to
 regenerate the Typescript client library from swagger. We use
 swagger-codegen-cli@2.4.7, which you can get
 [here](https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/2.4.7/).
-Make sure the jar file is somewhere on your path with the name
-swagger-codegen-cli.jar, then run `npm run apis`.
+Make sure the jar file is put in the frontend folder on your path with the name
```

Comment:
That is not correct, this jar file can put in anywhere as long as $PATH has the folder path that contains this jar.

You can provide the command which `add the jar file folder path to $PATH` here. So next time you can just run similar command to achieve the same goal.